<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWin</class>
 <widget class="QMainWindow" name="mainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>YouYuan</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tictactoeTitleLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>180</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YouYuan</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TIC TAC TOE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modeAivsAiButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>600</y>
      <width>1000</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NPC vs NPC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode1vsAiButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>660</y>
      <width>1000</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">align-content:center;</string>
    </property>
    <property name="text">
     <string>1 vs NPC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode1v1Button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>720</y>
      <width>1000</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1 vs 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="poweredByLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>721</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YouYuan</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> Powered By Artificial Intelligence (MiniMax Algorithm)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="chooseModeLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>1071</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose the Game mode you want to play:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>760</y>
      <width>991</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>YouYuan</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NPC- Non Player Character or (AI)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitGame">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>15</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Exit Game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
